### ISSUE TITLE
MailContact & EnableHybridMailFlow

### ISSUE THREAD

**Issue by adamdrayer** [timmcmic/DLConversionV2#23]
Hi Tim,

Not really an issue but more of a question.  

I understand enableHybridMailFlow creates a Dynamic Distribution Group with the mailcontact as the member, and the DDG is not synced so it can have the proxyaddresses of the original list.

My question is why not just have the mail contact with the same proxyaddreses as the original list and just not sync the mail contact object?  If it only exists on-prem and it has the original proxyaddreses - and the remote routing address of the new list is the targetaddress of the contact, wouldn't that allow for on-premises mail to be routed to the online list?

Thanks.

**Comment by timmcmic** [comment]
Adam:

Not a problem - here's the logic.  There needed to be a way to replace nested distribution group membership.  So on premises you migrate a sub DL.  Now - you want to email that parent DL - and expect that the subDL will be included.

So on premises it hits the parent DL -> hits the mail contact -> forward to DL in cloud -> expands and those recipients are included.

If we did not replicate the mail contact - we could not represent it in office 365.  You cannot replicate the mail contact and recycle the same proxy addresses - it causes a collision between the mail contact and the distribution group.

My blog series https://timmcmic.wordpress.com covers this and nested group membership.

It's complicated - but it works.

Tim

==============================
Timothy J. McMichael
Senior Support Escalation Engineer
***@***.******@***.***>
(980)-776-7465

Hours:  Sunday - Wednesday 08:00 - 16:00 eastern time zone.

Manager:  Tom Roughley ***@***.******@***.***>)

Premier Support - (800)-936-3100
Broad Commercial Support - (800)-936-4900

==============================

From: adamdrayer ***@***.***>
Sent: Thursday, October 7, 2021 11:28 AM
To: timmcmic/DLConversionV2 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [timmcmic/DLConversionV2] MailContact & EnableHybridMailFlow (#23)


Hi Tim,

Not really an issue but more of a question.

I understand enableHybridMailFlow creates a Dynamic Distribution Group with the mailcontact as the member, and the DDG is not synced so it can have the proxyaddresses of the original list.

My question is why not just have the mail contact with the same proxyaddreses as the original list and just not sync the mail contact object? If it only exists on-prem and it has the original proxyaddreses - and the remote routing address of the new list is the targetaddress of the contact, wouldn't that allow for on-premises mail to be routed to the online list?

Thanks.

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftimmcmic%2FDLConversionV2%2Fissues%2F23&data=04%7C01%7Ctimmcmic%40microsoft.com%7C5ffae9ddfa52421370a408d989a71699%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692173038425062%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=I1n3I78PtQYzKslG%2FIa2HtlpG3%2FSIQ43D3tyvLxVp9s%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAKGTN6NJZV66PWOUYNDA2CDUFW4BDANCNFSM5FRSDWZQ&data=04%7C01%7Ctimmcmic%40microsoft.com%7C5ffae9ddfa52421370a408d989a71699%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692173038425062%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=nluawHe61rcYnrmM3P2iztHuHlezqbisWSWOuJrIfeM%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Ctimmcmic%40microsoft.com%7C5ffae9ddfa52421370a408d989a71699%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692173038435055%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xOzHFgHb%2F73d9MAwhvEEk0X2sJ5UoEX%2BWiF7W%2B64wqo%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Ctimmcmic%40microsoft.com%7C5ffae9ddfa52421370a408d989a71699%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692173038445050%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=JoUOqnFCuq4NNc2cNau6H%2B45E%2FVANn4JaSxuWlCDAwQ%3D&reserved=0>.

**Comment by adamdrayer** [comment]
We are running hybrid.  We have migrated all our users but we have some IT-related monitoring systems that are on-prem and use the on-prem Exchange server as the target relay server.  So we have some groups that we migrated where we still have on-prem systems trying to email them.  After migrating the groups without the enableHybridMailFlow set to true, email stopped working, because there is no on-prem AD object that has the original proxyaddress of the group.  There is a mailcontact object, but the proxyaddress ends in "...-MigratedByScirpt@domain.com"
